.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
why we need this `fetch-depth: 0`?

Reply:
Isn't that needed for maven to run on the actual repository?

Reply:
no, afaik it is only needed for Sonar, we only use it in those jobs

Reply:
We also use it in the deploy workflow.